src/array.jl

```diff
@@ -773,6 +773,15 @@ Base.Broadcast.broadcasted(::typeof(!ismissing), A::CategoricalArray{T}) where {
     T >: Missing ? Base.Broadcast.broadcasted(>, A.refs, 0) :
                    Base.Broadcast.broadcasted(_ -> true, A.refs)
 
+function Base.Broadcast.broadcasted(::typeof(levelindex), A::CategoricalArray{T}) where {T}
+    ord = order(A.pool)
```

Comment:
why not `pool(A)`?

Reply:
The file mostly uses `A.pool`. Doesn't really matter I'd say, anyway this file by definition relies on the internal implementation.